public/static/locales/sv/common.json

```diff
@@ -1,5 +1,9 @@
 {
   "appName": "Kultus beta",
+  "pageMeta": {
+    "title": "Konst, kultur och evenemang för skolor och dagis",
+    "description": "Vi är ledsna, men det var ett fel med tjänsten. Du kan försöka igen, men om situationen kvarstår, vänligen meddela oss via feedbacklänken längst ner."
+  },
```

Comment:
There are changes done to Swedish common.json. How about Finnish and English? They seem to be empty at the moment.

Reply:
Oh yeah this sould be deleted, eneded up putting those texts to other file